src/vivarium_cluster_tools/psimulate/cluster/cli_options.py

```diff
@@ -7,10 +7,37 @@
 
 """
 
+from typing import List, Optional
+
 import click
 
 _RUNTIME_FORMAT = "hh:mm:ss"
 
+# https://docs.cluster.ihme.washington.edu/#hpc-execution-host-hardware-specifications
```

Comment:
Not sure of the best way to deal with the fact that the hardware changes. Alternatively we can not provide any constraints on what is passed in and leave it up to the user to "do their research"